apps/mull-api/src/app/channel/channel.resolver.ts

```diff
@@ -28,4 +35,21 @@ export class ChannelResolver {
   async getChannel(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
     return this.channelService.getChannel(id);
   }
+
+  @Query(/* istanbul ignore next */ () => EventChannel)
+  @UseGuards(AuthGuard)
+  async getEventChannel(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
+    return this.channelService.getEventChannel(id);
+  }
+
+  @Query(/* istanbul ignore next */ () => DirectMessageChannel, { nullable: true })
+  @UseGuards(AuthGuard)
+  async getDmChannel(
+    @Args('participants', { type: /* istanbul ignore next */ () => [UserInput] })
+    participants: UserInput[]
+  ) {
+    const [usr1, usr2] = participants;
+    const dmChannel = await this.channelService.findUsersInDmChannel(usr1.id, usr2.id);
+    return dmChannel ? dmChannel : null;
```

Comment:
This isn't secure. Since it's using AuthGuard, any user can query an DM channel by passing userId 1 and UserId 2.
We should use `AuthenticatedUser` for userId 1

Reply:
We want to make sure security is handled for all DM channel queries/mutations

Reply:
Thank you for bringing this up, we will check for security in our resolvers